docsrc/contributors/dynamo_converters.rst

```diff
@@ -0,0 +1,131 @@
+.. _dynamo_converters:
+
+Writing Dynamo Converters
+=============================
+The dynamo converter library in Torch-TensorRT is located in ``TensorRT/py/torch_tensorrt/dynamo/conversion``.
+
+Converter implementation
+------------------------
+
+Registration
+^^^^^^^^^^^^^^^^
+
+A converter is a function decrorated with  ``torch_tensorrt.dynamo.dynamo_tensorrt_converter`` that follows the function signature:
+
+
+.. code-block:: python
+
+    @torch_tensorrt.dynamo.conversion.dynamo_tensorrt_converter(torch.ops.aten.leaky_relu.default)
+    def leaky_relu_converter(
+        ctx: torch_tensorrt.dynamo.conversion.ConversionCtx,
+        target: Target,
+        args: Tuple[Argument, ...],
+        kwargs: Dict[str, Argument],
+        name: str,
+    ) -> Union[tensorrt.ITensor, Sequence[tensorrt.ITensor]]:
+
+
+
+The decorator takes a number of arguments:
+
+    * key: Node target for which the converter is implemented for (for example, torch.ops.aten.leaky_relu.default)
+    * enabled: Whether the converter should be enabled as a converter that can be used in the converter registry
+    * capability_validator: A lambda that can take a ``torch.fx.Node`` and determine if the converter can properly handle this Node. If the validator returns ``False``, the subgraph partitioner will make sure this Node is run in PyTorch in the compiled graph.
+    * priority: Allows developers to override existing converters in the converter registry
```

Comment:
Consider adding double backticks to each of the keyword arguments for clearer rendering